<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
font: 8pt &quot;Arial Rounded MT Bold&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_crear">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear la tabla Emails</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_solicitar">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>68</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Solicitar informacion de un Dominio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_consultar">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>130</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Consultar los datos de Emails</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_modificar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Modificar Correo de un Registro</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_eliminar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar un Registro</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_salir">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>370</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>230</y>
      <width>401</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Estos datos se mostraran despues de realizar una solicitud API </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>200</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Consulte nuevamente si modifico o elimino un registro</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEncontrar">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>108</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Encontrar correos electr√≥nicos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVerificar">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Verificar un correo electronico</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1059</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
